### ISSUE TITLE
Update labels: Issue Template: pre-work-template---ux

### ISSUE THREAD

**Issue by nnr-nnr** [hackforla/website#4484]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.
### Overview
As developers, we need to add the labels from the [Missing series](https://github.com/hackforla/website/wiki/How-to-read-and-interpret-labels#missing-series) so that we can avoid the GitHub bot taking off the labels when devs add them to issues.
### Action Items
- [x] Navigate to `.github/ISSUE_TEMPLATE/pre-work-template---ux.md`
- [x] Replace the following line
   ```
   labels: 'Feature: Onboarding/Contributing.md, prework, role: user research, size:1pt'
   ```
   with
   ```
   label: ['Feature: Onboarding/Contributing.md','prework','role: user research','size:1pt','Complexity: Missing']
   ```
- [ ] Test the updated template using these [instructions](https://github.com/hackforla/website/issues/4307#issuecomment-1549888931).  Copy the contents of that comment into a new comment in this issue, and check off the boxes as you complete the steps.  

**For PR Reviewers and Merge Team**
To review this issue, click the link below under "Link for reviewers" and verify that the labels match the updated labels string in the issue.

**Link for Reviewers**
This URL will be inserted by the dev during the test process and will be used by reviewers to preview the updated template.
URL of the issue branch on the test Repository:  [REPLACE THIS TEXT AND BRACKETS WITH THE URL] 

**For PM, Merge Team, or Tech Lead**
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #4307 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `.github/ISSUE_TEMPLATE/pre-work-template---ux.md`
- This issue is part of epic #4307

**Comment by arfgit** [comment]
Availability for this week: Thursday (May 11th)
My estimated ETA for completing this issue: 1 hour

**Comment by arfgit** [comment]
Provide Update
1. Progress - Completed
2. Blockers - No blockers
3. Availability - Thursday
4. ETA - 20 minutes

**Comment by ExperimentsInHonesty** [comment]
### Make an ER to add replace the labels on the Pre-work Template - UX issue template
````
- [ ] Replace 
```
label: ['Feature: Onboarding/Contributing.md','prework','role: user research','size:1pt','Complexity: Missing']
```
with
```
label: ['Feature: Onboarding/Contributing.md','prework','role: user research','size:1pt','Complexity: Prework']
```
````
